src/pages/recite-passage/recite-passage.html

```diff
@@ -1,13 +1,23 @@
-<!--
-  Generated template for the RecitePassagePage page.
-
-  See http://ionicframework.com/docs/components/#navigation for more info on
-  Ionic pages and navigation.
--->
 <ion-header>
   <ion-navbar>
     <ion-title>{{reference}}</ion-title>
+
+    <ion-buttons end>
+      <span class="btnPlayPause">
+        <button class="btnPlayPause__icon" ion-button (click)="onAudioToggle()" aria-label="Listen to the passage">
+          <ion-icon name="{{ playPauseIcon }}"></ion-icon>
+        </button>
+
+        <svg height="36" version="1.1" viewBox="0 0 100 100" width="36" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Seems a bit ugly to include the `SVG` this way… I wonder if there is a better way with Ionic?

Reply:
I'm not going to fix this here.